### ISSUE TITLE
Better styling

### ISSUE THREAD

**Issue by brandonchinn178** [brandonchinn178/circle-of-suck#42]
styled-components?

**Comment by shughes26** [comment]
CMB alum here, anything specific you might want for this?

**Comment by brandonchinn178** [comment]
Hi Sarah! Really, anything is better than now :)

Thinking about it more, maybe we should just implement our own graph instead of using `react-graph-vis`. I avoided doing this for now because it's a ton of work that I don't have a lot of experience in, but making the graph look nicer and more interactive (e.g. https://github.com/brandonchinn178/circle-of-suck/issues/44) would probably up the production value of the site a ton.

If you have experience with d3, then maybe you can look into rendering the circle of suck (currently stored in `circleOfSuckEdges`) as a d3 graph.

If you also have experience designing, then it would be great even to just get a mock up of what we'd like the circle of suck to look like

**Comment by shughes26** [comment]
Yeah I'd be happy to help with finding a better way to render the graph. I've never used d3 specifically, but I'm sure I could figure it out. We could also look into what Highcharts might have available from their open source stuff.

https://www.christophermanning.org/projects/building-cubic-hamiltonian-graphs-from-lcf-notation <-- (edit) this could be interesting

**Comment by brandon-leapyear** [comment]
:sparkles: _**This is an old work account. Please reference @brandonchinn178 for all future communication**_ :sparkles:


---

ooh that does look cool! Ideally, we'd want to allow some user interaction, so we'd need to make sure that a third party library would support it.

Highcharts looks interesting. I think the problem with data visualization libraries (like d3 and highcharts) are that they're focused on data charts, instead of rendering a graph structure. But maybe one of those can render graphs!

**Comment by shughes26** [comment]
They both seem like they have graph support as long as you already know what the nodes/edges are supposed to be

**Comment by brandonchinn178** [comment]
Awesome. Go ahead and use whatever you're comfortable with! I'll be happy to look at a PR
